evm/p2p/p2p_proto.py

```diff
@@ -20,18 +20,16 @@ class Hello(Command):
         ('remote_pubkey', sedes.binary)
     ]
 
-    @classmethod
-    def handle(cls, proto, data):
-        hello = cls.decode(data)
+    def handle(self, proto, data):
+        hello = self.decode(data)
         return hello
 
 
 class Disconnect(Command):
-    id = 1
+    _id = 1
```

Comment:
That was its original name, but I renamed it  to id because it's an attribute of a class named `Command`, so it should be clear enough. What do you think?

Reply:
That was sort of what I assumed was going on.  Two thoughts.

* `cmd.cmd_id` is better than `cmd._id` in my opinion (it's at least not worse).
* Most commands have other names so it would be things like `status.cmd_id` or `hello.cmd_id` so I think the prefix is fine.

Reply:
Fair enough, but this will be _cmd_id because the public attribute (currently `id`, now renamed to `cmd_id`) is a `@property`